app/src/main/java/com/epfl/drawyourpath/database/MockDataBase.kt

```diff
@@ -31,13 +80,28 @@ class MockDataBase : Database() {
 
     override fun setUserGoals(
         username: String,
-        distanceGoal: Int,
-        timeGoal: Int,
+        distanceGoal: Double,
+        timeGoal: Double,
         nbOfPathsGoal: Int
     ) {
         val userGoals: UserGoals = UserGoals(username, distanceGoal, timeGoal, nbOfPathsGoal)
         usernameToUserGoalsDataBase.put(username, userGoals)
     }
+
```

Comment:
It will refactor this class during theme next task, so  will probably delete this method to sendall the PersoInfo and the user goals in a same transaction.